tests/images.spec.ts

```diff
@@ -18,3 +28,60 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImage(page, instance);
+
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+  page.getByText(imageAlias);
+  const imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
```

Comment:
```suggestion
    `text=was successfully downloaded. Please check your downloads folder.`,
```
Maybe we can save the extra complexity to figure out the image name and rely on delete all images instead?

Reply:
Won't deleteAllImages give us the error with the my-custom-image, again?

Reply:
Yes, absolutely right, lets keep this as is. When I wrote this comment I didn't know yet about that problem.